eng/Build.props

```diff
@@ -1,8 +1,17 @@
 <?xml version="1.0" encoding="utf-8"?>
 <Project>
+
+  <ItemDefinitionGroup>
+    <ProjectToBuild>
+      <RestoreInParallel>true</RestoreInParallel>
+      <BuildInParallel>false</BuildInParallel>
+    </ProjectToBuild>
```

Comment:
question: Where does it fail if we enable parallel builds?

Reply:
I'll run it again and see.

Reply:
Various things fail. The native .dll hasn't finished compiling by the time its needed in the other .cs projects, so the copy fails. When the API checker runs, some of the c# .dll's aren't done yet so it fails cause the contracts is different. etc. Once its been fully built once, it works fine in parallel. But the first time it can't be. But the parallel build is just for the different main projects in the build.props file. It doesn't affect the compilation of ml.net itself. At least as far as I can tell.

Reply:
Parallel builds should respect dependencies. If it is failing, it looks like we are not capturing the project dependencies correctly. Slightly lower priority work item, but we should see where it is failing. 
Can you please add either a TODO or an explicit bug for this so we don't forget this?


---
In reply to: [513684740](https://github.com/dotnet/machinelearning/pull/5464#discussion_r513684740) [](ancestors = 513684740)

Reply:
Lets sync on this. I'm not sure that its a dependency issue.

Also, the API checker has issues even not in parallel if you run PACK with the first build. You need to run a full build without `-pack` the first time. This is probably a dependency issue, but I would like to sync on that with you briefly as well.

Reply:
Resolved per offline discussion.